arch/Makefile.archer2_openmpi+openmp

```diff
@@ -0,0 +1,55 @@
+# HQ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# HQ X
+# H0 X   libAtoms+QUIP: atomistic simulation library
+# HQ X
+# HQ X   Portions of this code were written by
+# HQ X     Tamas K. Stenczel
+# HQ X
+# HQ X   Copyright 2021
+# HQ X
+# HQ X
+# HQ X   Edited by Vlad Carare and Lars Schaaf - working as of 07.04.2022
+# HQ X
+# HQ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+# ACRHER2 computer cluster, United Kingdom
+#
+# recommended modules to use with this:
+# - GNU compilers version 9, python and FFTW
+# - obtain these with the following on Archer2
+#
+# module switch PrgEnv-cray PrgEnv-gnu
+# module load PrgEnv-gnu
+# module load gcc/9.3.0
+# module load cray-fftw
+
+
+
+# declarations
+
+include arch/Makefile.linux_x86_64_gfortran
+
+# compiler settings, make sure this is GNU!
+F77 = ftn
+F90 = ftn
+F95 = ftn
+CC = cc
+CPLUSPLUS = cc
+LINKER = ftn
+
+# OpenMP
+DEFINES += -D_OPENMP
```

Comment:
You can remove this line to get rid of `Warning: "_OPENMP" redefined`. Most compilers add this macro if you select the openmp option, that's why the warning is given when you declare it again manually.

Reply:
Thanks! Modified